src/subdomains/supporting/dex/services/dex-bitcoin.service.ts

```diff
@@ -10,17 +12,44 @@ import { LiquidityOrderRepository } from '../repositories/liquidity-order.reposi
 export class DexBitcoinService {
   #client: BtcClient;
 
-  constructor(private readonly liquidityOrderRepo: LiquidityOrderRepository, readonly nodeService: NodeService) {
+  constructor(
+    private readonly liquidityOrderRepo: LiquidityOrderRepository,
+    private readonly feeService: BtcFeeService,
+    readonly nodeService: NodeService,
+  ) {
     nodeService.getConnectedNode(NodeType.BTC_OUTPUT).subscribe((client) => (this.#client = client));
   }
 
+  async sendUtxoToMany(payout: { addressTo: string; amount: number }[]): Promise<string> {
+    const feeRate = await this.feeService.getRecommendedFeeRate();
+    return this.#client.sendMany(payout, feeRate);
+  }
+
+  async transferMinimalUtxo(address: string): Promise<string> {
+    return this.sendUtxoToMany([{ addressTo: address, amount: Config.blockchain.default.minDeposit.Bitcoin.BTC / 2 }]);
+  }
```

Comment:
Is this used? `Config.blockchain.default.minDeposit.Bitcoin.BTC / 2` is currently more than 5 USD 😉

Reply:
/ 1000 instead